specs/signature-specification.md

```diff
@@ -1,60 +1,18 @@
 # Signature Specification
 
-This document provides the following details for Notary v2 signatures:
+This document provides the following details for Notary Project signatures:
 
 - **[Storage](#storage)**: Describes how signatures are stored and retrieved from an OCI registry.
-- **[Signature Envelope](#signature-envelope)**: Describes the structure of a Notary v2 signature.
+- **[Signature Envelope](#signature-envelope)**: Describes the structure of a Notary Project signature.
 
 ## Storage
 
-This section describes how Notary v2 signatures are stored in the OCI Distribution conformant registry.
-Notary v2 uses [OCI artifact manifest][oci-artifact-manifest] to store signatures in the registry.
-The media type of the signature manifest is `application/vnd.oci.artifact.manifest.v1+json`.
-The signature manifest has an artifact type that specifies it's a Notary V2 signature, a reference to the manifest of the artifact being signed, a blob referencing the signature, and a collection of annotations.
+This section describes how Notary Project signatures are stored in the OCI Distribution conformant registry.
+OCI image manifest is used to store signatures in the registry, see [OCI image spec v1.1.0-rc.2][oci-image-manifest] for details.
```

Comment:
Why should we refer to the OCI 1.1.0-rc2 image spec here. Can't we refer to the OCI 1.0 image spec here?

Reply:
Yes, we can use OCI 1.0 image spec. but the difference is that notary signature uses additional optional field named subject for referring to signed artifacts, which is described in OCI 1.1.